src/Carousel.svelte

```diff
@@ -9,7 +9,7 @@
     {#if dots}
 	<ul>
 		{#each pips as pip, i}
-		<li on:click={() => go(i)} class={currentIndex === i ? "active" : ""}></li>
+		<li on:click={() => go(i)} class={currentIndex === i || (currentIndex<0 && currentIndex == i-controller.innerElements.length) ? "active" : ""}></li>
```

Comment:
I think fixing the way that active is displayed isn't quite the correct approach here.

The way I would suggest this is fixed is to rectify `currentIndex` if it goes below 0.

Also note you should always use triple equals (strict equality) for comparisons